src/gui/SendButtonIndicator.cpp

```diff
@@ -2,32 +2,30 @@
 
 #include "embed.h"
 #include "Mixer.h"
-#include "MixerLine.h"
+#include "MixerChannelView.h"
 #include "MixerView.h"
 
 namespace lmms::gui
 {
 
-SendButtonIndicator:: SendButtonIndicator( QWidget * _parent, MixerLine * _owner,
-										   MixerView * _mv) :
-	QLabel( _parent ),
-	m_parent( _owner ),
-	m_mv( _mv )
+SendButtonIndicator:: SendButtonIndicator(QWidget * parent, MixerChannelView * owner, MixerView * mv) :
+	QLabel(parent),
+	m_parent(owner),
+	m_mv(mv)
 {
-
-	// don't do any initializing yet, because the MixerView and MixerLine
+	// don't do any initializing yet, because the MixerView and MixerChannelView
 	// that were passed to this constructor are not done with their constructors
 	// yet.
 	setPixmap(m_qpmOff);
 }
 
-void SendButtonIndicator::mousePressEvent( QMouseEvent * e )
+void SendButtonIndicator::mousePressEvent(QMouseEvent * e)
```

Comment:
Extra space between type and `*`

Reply:
Last I mentioned this in a PR, they said it wasn't in our conventions? Though, yes, I do prefer no space.

Reply:
The reason I mention it is because it's inconsistent in the file. I'd rather the whole file be consistent with format. You're correct though that we have no convention for that, but I believe we do have it in the clang-tidy formatter. *Technically* since we fall back to Qt convention first, they actually pair the `*` with the variable name; but the C++ Core Guidelines shows pairing with the type. We tend towards pairing with type.

Reply:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#nl15-use-spaces-sparingly
Guidelines showing pairing with type

https://wiki.qt.io/Qt_Coding_Style
Qt showing pairing with name

Reply:
I strongly prefer pairing with type since it makes it much easier and faster to visually parse without context by adding this consistency:

```cpp
foo* bar // always a pointer
foo * bar // always multiplication
foo *bar // always a dereference

foo& bar // always a reference
foo & bar // always bit-wise AND
foo &bar // always address-of
```

Reply:
Fixed with commit 92878ce4030. Using reference/pointer at type like in other places in the code.